wpimath/src/main/native/include/frc/geometry/struct/Ellipse2dStruct.h

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#pragma once
+
+#include <wpi/SymbolExports.h>
+#include <wpi/struct/Struct.h>
+
+#include "frc/geometry/Ellipse2d.h"
+
+template <>
+struct WPILIB_DLLEXPORT wpi::Struct<frc::Ellipse2d> {
+  static constexpr std::string_view GetTypeString() {
+    return "struct:Ellipse2d";
+  }
+  static constexpr size_t GetSize() {
+    return wpi::GetStructSize<frc::Pose2d>() + 16;
+  }
+  static constexpr std::string_view GetSchema() {
+    return "Pose2d center;double xSemiAxis;double ySemiAxis";
+  }
+
+  static frc::Ellipse2d Unpack(std::span<const uint8_t> data);
+  static void Pack(std::span<uint8_t> data, const frc::Ellipse2d& value);
+  static void ForEachNested(
+      std::invocable<std::string_view, std::string_view> auto fn) {
+    wpi::ForEachStructSchema<frc::Pose2d>(fn);
+    wpi::ForEachStructSchema<units::meter_t>(fn);
+    wpi::ForEachStructSchema<units::meter_t>(fn);
```

Comment:
Is there a struct implementation for `units::meter_t`?

Reply:
I have no idea how any of this serialization stuff works. I was just blindly copy-pasting.

Reply:
It doesn't need to nest into basic values (these just get serialized as double values), so these two lines can be deleted.